sdk/storage/storage-blob/src/BlobServiceClient.ts

```diff
@@ -173,6 +173,54 @@ export class BlobServiceClient extends StorageClient {
     });
   }
 
+  /**
+   * Iterates over containers under the specified account.  The returned iterator is resumable.
+   *
+   * @example
+   * for await (const container of blobServiceClient.listContainers()) {
+   *   console.log(`Container: ${container.name}`);
+   * }
+   *
+   * @example
+   * let iter1 = blobServiceClient.listContainers();
+   * let i = 1;
+   * for await (const container of iter1) {
+   *   console.log(`${i}: ${container.name}`);
+   *   i++;
+   * }
+   *
+   * @example
+   *  let iter2 = await blobServiceClient.listContainers();
+   *  i = 1;
+   *  let containerItem = await iter2.next();
+   *  do {
+   *    console.log(`Container ${i++}: ${containerItem.value.name}`);
+   *   containerItem = await iter2.next();
+   * } while (containerItem.value);
+   *
+   * @param {ServiceListContainersSegmentOptions} [options]
```

Comment:
Add some comments for `[options]`, "Optional options to list container segment" -- feels a bit weird as it doesn't list segment though. maybe just "Optional options to list container"?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/3136/commits/72cd066b00911e8ff89e914e6864fb860ff1c3d1